main.py

```
@@ -202,44 +234,61 @@ def copy_to_clipboard(text: str) -> ResultResponse:
+            print(f"Error: Text editor not found: {text_editor}")
+            return
+
         # Create a temporary file with the text content
         fd, path = tempfile.mkstemp(suffix=".txt", prefix="ai_response_")
         with os.fdopen(fd, 'w', encoding='utf-8') as f:
             f.write(text)
-        
+
         # Open the file with notepad using subprocess
-        subprocess.Popen(["notepad.exe", path])
+        subprocess.Popen([text_editor, path])
```

Ruff (0.14.0)

`subprocess` call: check for execution of untrusted input

(S603)